packages/ubuntu_desktop_installer/lib/installer.dart

```diff
@@ -8,6 +8,7 @@ import 'package:path/path.dart' as p;
 import 'package:subiquity_client/subiquity_client.dart';
 import 'package:subiquity_client/subiquity_server.dart';
 import 'package:timezone_map/timezone_map.dart';
+import 'package:ubuntu_desktop_installer/services/canberra_service.dart';
```

Comment:
could you `export 'services/canberra_service.dart'` in the `services.dart` barrel import file (which is already imported below)